DESCRIPTION

```diff
@@ -11,15 +11,21 @@ Authors@R: c(person("Jon","Clayden",role=c("cre","aut"),email="code@clayden.org"
              person("Jean-loup","Gailly",role="cph"),
              person("Mark","Adler",role="cph"))
 Imports: Rcpp (>= 0.11.0)
-Suggests: testthat (>= 0.11.0), covr, reportr, shades
-Enhances: oro.nifti, tractor.base
+Suggests: 
+    testthat (>= 0.11.0),
+    covr,
+    reportr,
+    shades,
+    oro.nifti,
+    tractor.base
```

Comment:
The tests using `oro.nifti` and `tractor.base` are conditional on the presence of those packages, as required by Writing R Extensions, so leaving them in `Enhances` should be fine. Plus, they both have hard dependencies on `RNifti`, so this should be as weak as possible to avoid circularity.